src/main/webapp/view/discussion/discussion-area.js

```diff
@@ -12,13 +12,13 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import {ScrollContainer} from '../../scroll-container.js';
 import {timestampToString} from '../../timestamps.js';
+
```

Comment:
Is this new line necessary or is it due to the linter?

Reply:
got rid of it